test/components.tsx

```diff
@@ -166,8 +156,8 @@ test('squash multiple nested text nodes', t => {
 
 test('squash empty `<Text>` nodes', t => {
 	const output = renderToString(
-		<Box unstable__transformChildren={string => `[${string}]`}>
-			<Box unstable__transformChildren={string => `{${string}}`}>
+		<Box unstable__transformChildren={(str: string) => `[${str}]`}>
+			<Box unstable__transformChildren={(str: string) => `{${str}}`}>
```

Comment:
Don't use abbreviations for variable names.
```suggestion
			<Box unstable__transformChildren={(string: string) => `{${string}}`}>
```

Reply:
I don't usually but there are 6 other places in this file that use `str` as an argument to `unstable__transformChildren` prop. I changed it here to match. I'll remove the abbreviation in the other 6 places.